ephios/core/templatetags/settings_extras.py

```diff
@@ -1,16 +1,34 @@
+from urllib.parse import urljoin
+
 from django import template
 from django.conf import settings
-
-from ephios.core.views.settings import get_available_management_settings_sections
+from dynamic_preferences.registries import global_preferences_registry
 
 register = template.Library()
 
 
 @register.simple_tag
 def available_management_settings_sections(request):
+    from ephios.core.views.settings import get_available_management_settings_sections
+
     return get_available_management_settings_sections(request)
 
 
 @register.simple_tag
 def oidc_client_enabled():
     return settings.ENABLE_OIDC_CLIENT
+
+
+@register.simple_tag
+def site_url():
+    return settings.GET_SITE_URL()
+
+
+@register.simple_tag
+def organization_name():
```

Comment:
is there any benefit of having a tag instead of inserting it into the base context?

Reply:
I think it's mainly because the base context was only available in the request response cycle and e.g. not in E-Mails. I also think the tag is more explicit.